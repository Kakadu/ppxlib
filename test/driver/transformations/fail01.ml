type t =
  { b : int
  ; a : int
  }
