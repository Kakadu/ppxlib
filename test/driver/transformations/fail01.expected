[@@@ocaml.ppwarning "Fields are not sorted!"]
type t = {
  b: int ;
  a: int }
